cnxarchive/search.py

```diff
@@ -37,6 +37,7 @@
                  [chr(i) for i in range(ord('a'), ord('z') + 1)])
 WILDCARD_KEYWORD = 'text'
 VALID_FILTER_KEYWORDS = ('type', 'pubYear', 'authorID', 'submitterID')
+SEARCH_SPECIAL_CHARS = '"\',:'
```

Comment:
This won't affect how searches are done. Stripped, `don't` will be `dont` which is a valid search.